client/app/bundles/course/assessment/containers/AssessmentForm.jsx

```diff
@@ -229,14 +230,21 @@ class AssessmentForm extends React.Component {
         <Field
           name="autograded"
           component={Toggle}
-          label={<FormattedMessage {...translations.autograded} />}
+          label={
+            modeSwitching ? <FormattedMessage {...translations.autograded} /> :
+            <FormattedMessage {...translations.modeSwitchingDisabled} />
```

Comment:
Is this only shown if the assessment is autograded? If this is show for manually-graded assessments we might need to change the message.